<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMusicWindow</class>
 <widget class="QMainWindow" name="SearchMusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>141</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>GSP - Musik suchen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#cmbAddAll {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_normal.png);
padding-right: 25px;
padding-left: 10px;
}

#cmbAddAll::drop-down {
border-width: 0px;
}

#cmbAddAll:hover {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_hover.png);
}

#cmbAddAll:on {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_pressed.png);
}

#cmbAddAll:disabled {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Dropdown_disabled.png);
}

#btnBack {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Zurueck_pressed.png);
border: none;
background-repeat: none;
}

#btnBack:hover {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Zurueck_hover.png);
}

#btnBack:pressed {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Zurueck_normal.png);
}

#lblBtnBack {
font: 14px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255,255,255);
}

#lblSearchMusic {
font: 16px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255, 255, 255);
}

QLabel {
font: 14px &quot;Arial&quot;;
color: rgb(163, 163, 163);
}

#txtSearchSong {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Input.png);
padding: 0 6px;
}

#txtSearchArtist {
border-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Input.png);
padding: 0 6px;
}

#lblSearchSong, #lblAddAll {
font: 14px;
font-weight: bold;
font-family: arial, verdena, sans-serif;
color: rgb(255, 255, 255);
}

#lblSearchArtist {
font: 14px;
font-weight: bold;
font-family: arial, verdena, sans-serif;
color: rgb(255, 255, 255);
}

#btnPopular, #btnSearchSong, #btnSearchArtist {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Search_normal.png);
border: none;
background-repeat: none;
font: 14px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255,255,255);
}

#btnAddAll {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_normal.png);
border: none;
background-repeat: none;
font: 14px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255,255,255);
}

#btnPopular:hover, #btnSearchSong:hover, #btnSearchArtist:hover {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Search_hover.png);
}

#btnAddAll:hover {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_hover.png);
}

#btnAddAll:pressed {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Download_Button_pressed.png);
}

#btnPopular:pressed, #btnSearchSong:pressed, #btnSearchArtist:pressed {
background-image: url(:/theme/searchmusic/RES/theme/Musik_Suche/Search_pressed.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>383</width>
      <height>141</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="Design.qrc">:/theme/searchmusic/RES/theme/Musik_Suche/BG_Top.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>0</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblBtnBack">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>4</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>zurück</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSearchMusic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>106</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Musik suchen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSearchSong">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>60</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Suchen:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearchSong">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>50</y>
      <width>162</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearchSong">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>50</y>
      <width>76</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Suche</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstItems">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>141</y>
      <width>383</width>
      <height>459</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPopular">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>94</y>
      <width>76</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Populär</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAddAll">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>103</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alle hinzufügen zu:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbAddAll">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>94</y>
      <width>163</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Design.qrc"/>
 </resources>
 <connections/>
</ui>
